src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classNames from 'classnames';
+import PopperJS, { Data, ModifierFn } from 'popper.js';
+import { Manager, Popper, Reference } from 'react-popper';
+
+export interface TooltipProps {
+	children: ReactNode;
+	index: string;
```

Comment:
Kun je dit niet gewoon `id` noemen? Index duidt eerder op een volgorde in een lijst.